hwdb.d/60-sensor.hwdb

```diff
@@ -529,6 +533,12 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:rvnPOV:rnI102A:*
 sensor:modalias:i2c:bmc150_accel:dmi:bvnINSYDECorp.:*:svnInsyde:pnBayTrail:*:rvn105B:rn0E57:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
+#########################################
+# Purism
+#########################################
+sensor:modalias:of:NproxT*Cvishay,vcnl4040Cvishay,vcnl4200:*
+ PROXIMITY_NEAR_LEVEL=220
```

Comment:
Can you split the portion of the patch that relates to the documentation additions from adding the device?

Reply:
yes, i dropped the device completely as per above discussion